fabtests/include/windows/unistd.h

```diff
@@ -1,3 +1,16 @@
 #pragma once
 
 #define sleep(x) Sleep(x * 1000)
+
+static inline void usleep(DWORD waitTime)
+{
+	LARGE_INTEGER perfCnt, start, now;
+
+	QueryPerformanceFrequency(&perfCnt);
+	QueryPerformanceCounter(&start);
+
+	do {
+		QueryPerformanceCounter((LARGE_INTEGER*)&now);
+	} while ((now.QuadPart - start.QuadPart) /
+		(float)perfCnt.QuadPart * 1000 * 1000 < waitTime);
+}
```

Comment:
This isn't really sleeping, so may not relinquish the cpu to the other thread for processing.  Can we just divide the time by 1000, rounding up, and call Sleep()?  We don't really care if usleep takes a full ms, since we can't really control that.

Reply:
Your observation is correct, but I pulled this directly from include/windows/unistd.h assuming that what was good for libfabric was good for fabtests.

Would you like a PR to change this in the libfabric include as well?

Reply:
Yes, please, fixing libfabric would be great.

Reply:
I'll create a PR for the same code in libfabric once you've approved the changes here.